platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/AssertionConsumerService.java

```diff
@@ -200,11 +200,16 @@ public Response getSamlResponse(
       @QueryParam(SIG_ALG) String signatureAlgorithm,
       @QueryParam(SIGNATURE) String signature) {
 
-    LOGGER.info("HTTP-Redirect binding should not be used for Single Sign On responses");
+    LOGGER.warn("HTTP-Redirect binding should not be used for Single Sign-On responses");
     if (validateSignature(deflatedSamlResponse, relayState, signatureAlgorithm, signature)) {
       try {
-        return processSamlResponse(
-            RestSecurity.inflateBase64(deflatedSamlResponse), relayState, signature != null);
+        return (StringUtils.isNotBlank(deflatedSamlResponse)
+                && StringUtils.isNotBlank(relayState)
```

Comment:
✏️ Might be a good thing to annotate the method parameters that can be `null` with `@Nullable` too such that it be clear.

Reply:
The only parameter that doesn't throw an error when `null` is `signatureAlgorithm` and prints `"Received unsigned AuthN response.` to the logs. In light of that, I'm not going to make it explicit that it's `@Nullable`